model/cookies-preference.go

```diff
@@ -1,9 +1,10 @@
 package model
 
-import "github.com/ONSdigital/dp-renderer/model"
+import "github.com/ONSdigital/dp-renderer/v2/model"
 
 // CookiesPreference is the model struct for the cookies preferences form
 type CookiesPreference struct {
 	model.Page
-	PreferencesUpdated bool `json:"preferences_updated"`
+	PreferencesUpdated bool                `json:"preferences_updated"`
+	TypeRadios         model.RadioFieldset `json:"type_radios"`
```

Comment:
Because I didn't understand the context of it when I was lifting the code from the feedback controller. 

I thought perhaps TypeRadios was a reference to it name as aclass or struct. 

I can see from your comment that TypeRadios is just the name you gave it and what I'm referring to is more than likely RadioFieldset.

Reply:
You're right it's `name` then `type`